tests/tests/legacy_kustomizations/knative-install/test_data/expected/apps_v1_deployment_networking-istio.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       app.kubernetes.io/part-of: kubeflow
       app.kubernetes.io/version: v0.8.0
       kustomize.component: knative
+      serving.knative.dev/release: v0.11.1
```

Comment:
if everything is changing to 0.11.2 - this should as well?

Reply:
ye let's change all to 0.11.2, I looked at the difference between 0.11.1 and 0.11.2, there is NO manifest change but all image sha are changed due to image rebuild I think.

Reply:
K